languages/wikipron/src/whitelist.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+import sys
+import re
+
+
+def whitelist_reader(path):
+    """Reads in whitelist file"""
+    with open(path, "r") as source:
+        for line in source:
+            yield line.rstrip().split()[0]
+
+
+def filter(tsv_path, phonemes, output_path):
+    """Creates tsv filtered by whitelist"""
+    with open(tsv_path, "r") as source:
+        with open(output_path, "w") as output:
+            for line in source:
+                pron = set(line.split("\t")[1].split())
+                if phonemes.issuperset(pron):
+                    print(line.rstrip(), file=output)
+                else:
+                    pass
+
+
+def main() -> None:
+    tsv_path = sys.argv[1]
```

Comment:
This will crash unceremoniously if someone doesn't provide a file as the command-line argument. Use `argparse` instead, like we did for the `evaluate_all.py` script you wrote and put the argument parsing inside `if __name__ == __main__:`, passing it to `main`.